README.md

```diff
@@ -10,9 +11,11 @@
 
 ## 📦 Installation
 
+This package is independent of Vapor and can be used for all Swift projects up til version 4.2
+
 Update your `Package.swift` file.
 ```swift
-.Package(url: "https://github.com/nodes-vapor/slugify", majorVersion: 1)
+.package(url: "https://github.com/nodes-vapor/slugify", from: "1.0.0")
```

Comment:
Let's put this at `2.0.0` and then do a new major to be on the safe side (since we're bumping the Swift major version).